examples/http-response-interceptor/http_response_interceptor.md

```diff
@@ -2,10 +2,14 @@
 
 The `http:ResponseInterceptor` is used to intercept the response and execute custom logic. A `ResponseInterceptor` is a service object with a remote method called `interceptResponse`, which is executed before dispatching the response to the client. A `ResponseInterceptor` can be created from a service class, which includes the `http:ResponseInterceptor` service type. 
 
-This service object can be engaged at the listener level by using the `interceptors` field in the `http:ListenerConfiguration` or at the service level by declaring a `http:InterceptableService` object. These accept an interceptor service object or an array of interceptor service objects as an interceptor pipeline and the interceptors are executed in the order in which they are placed in the pipeline. 
+This service object can be engaged at the service level by declaring a `http:InterceptableService` object. This accept an interceptor service object or an array of interceptor service objects as an interceptor pipeline and the interceptors are executed in the order in which they are placed in the pipeline. 
 
 Use `ResponseInterceptors` to execute common logic such as logging, header manipulation, state publishing, etc., for all outbound responses.
 
+When handling `Resource/Service Not Found` scenarios, if there is only a single service attached to the listener, the interceptor pipeline 
```

Comment:
@shafreenAnfar do we need to mention this in a BBE?
@dilanSachi we have added this in the spec right?

Reply:
Yup. It is added in the spec.